### PR TITLE
Task #53365 fix - Created Add page and See all pages

### PR DESCRIPTION
**Pull request by CosminaX**

### CODE REVIEW

**Review by petricafinaga — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -14,6 +18,15 @@ function App() {
   return (
     <div className="App">
       <BrowserRouter>
+        <Layout>
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/location" element={<Location />} />
+            <Route path="/SeeAllPages" element={<SeeAllPages />} />
+            <Route path="/add" element={<Add />} />
+            {/* <Route path="/*" element={<Navigate replace to="/" />} /> */}
```
Comment (petricafinaga) [inline src/App.js:27]: We need this?

File: src/Components/API.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+//Get user by id
+export const GetUserById = async (id) => {
+  try {
+    const response = await axios.get(
+      "https://assist-august-2022-be2.azurewebsites.net/api/user/" + id + "/"
```
Comment (petricafinaga) [inline src/Components/API.js:34]: The base url of the endpoint can be used from the .env file like we are doing for other api's

File: src/app.scss [hunk 3]
```diff
@@ -1,7 +1,6 @@
-body{
-    margin:0 ;
-    padding:0;
-    overflow: hidden;
-    font-family:'Inter';
+body {
+  margin: 0;
+  padding: 0;
+  // overflow: hidden;
```
Comment (petricafinaga) [inline src/app.scss:4]: need this?

File: src/index.js [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  // <React.StrictMode>
```
Comment (petricafinaga) [inline src/index.js:8]: need this?

File: src/pages/Add/Add.js [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+          "FILE_FORMAT",
+          "Uploaded file has unsupported format.",
+          (value) =>
+            !value || (value && SUPPORTED_FORMATS.includes(value?.type))
+        ),
+    }),
+
+    onSubmit: async (values) => {
+      try {
+        // const response = await createListing(values);
```
Comment (petricafinaga) [inline src/pages/Add/Add.js:59]: need this?

File: src/pages/homepage/HomePage.js [hunk 6]
```diff
@@ -1,27 +1,88 @@
...
 
 function HomePage() {
+  // const [user, setUser] = useState();
+  // const fetchUser = async () => {
+  //   try {
+  //     //get userId from local storage
+  //     const response = await GetUserById(id);
+  //     setUser = response.data;
+  //   } catch (error) {
+  //     console.log(error);
```
Comment (petricafinaga) [inline src/pages/homepage/HomePage.js:22]: need all this commented info?

File: src/pages/homepage/HomePage.js [hunk 7]
```diff
@@ -41,30 +102,170 @@ function HomePage() {
...
+                />
+              </button>
+            </div>
+            <div className="content">
+              <Slider {...settings} className="card__container--inner">
+                {homes.map((home, index) => (
+                  <div key={home.pk}>
+                    <Cards
+                      title={home.title}
+                      // location={home.location}
```
Comment (dianamiraute) [inline src/pages/homepage/HomePage.js:172]: Pay attention at commented code

**Review by dianamiraute — APPROVED** [review]

**Review by BiancaDianaAlexiuc — APPROVED** [review]

**Pull request merged**